vnda/loaders/productListingPage.ts

```diff
@@ -63,31 +65,32 @@ const searchLoader = async (
   const term = props.term || props.slug || qQueryString ||
     undefined;
 
+  const categoryTagName = props.term || url.pathname.split("/").pop() || "";
+
+  const categoryTag = isSearchPage
+    ? undefined
+    : await api["GET /api/v2/tags/:name"]({ name: categoryTagName }, STALE)
+      .then((res) => res.json()).catch(() => undefined);
+
   const response = await api["GET /api/v2/products/search"]({
     term,
     sort,
     page,
     per_page: count,
```

Comment:
To thos two calls at same time.

```suggestion
  const categoryTagPromise = isSearchPage
    ? Promise.resolve(undefined)
    : api["GET /api/v2/tags/:name"]({ name: categoryTagName }, STALE)
      .then((res) => res.json()).catch(() => Promise.resolve(undefined));

  const [categoryTag, response]= await Promise.all(categoryTagPromise, api["GET /api/v2/products/search"]({
    term,
    sort,
    page,
    per_page: count,
```

Reply:
ignore it, cannot do categoryTag in paralel.